cbor-file-magic.mkd

```diff
@@ -168,8 +174,10 @@ The most obvious is that it may allow malware to identify interesting objects on
 
 # IANA Considerations
 
-There are no IANA actions.
-This section documents the allocation that was done.
+{{cbor-sequence-tag}} documents the allocation that was done for a
```

Comment:
Seems weird to have a forward reference in what is normally a closing section of an RFC.
Should we change the order of sections?

Reply:
This is not a "forward reference".  Sections often start with saying what is in the section.  Here, we have links to the two subsections.  I don't know what exactly irks you here.